app/assets/stylesheets/custom.scss.erb

```diff
@@ -5084,3 +5084,78 @@ input:required:valid {
   margin-top: -2rem;
 }
 
+.carrier-filter{
+  h4 {
+    font-weight: bold;
+  }
+
+  .row {
+    h5 {
+      font-size: 1.4rem;
+      font-weight: 400;
+      margin-bottom: 10px;
+    }
+    .form-group {
+      label {
+        font-size: 1.5rem;
+      }
+
+      input[type="checkbox"] {
+        margin-right: 8px;
+        margin-top: -2px;
+        vertical-align: middle;
+        line-height: normal;
+        width: 20px;
+        height: 20px;
+      }
+    }
+  }
+
+  #search-input-box{
+    display: flex;
+    align-items: center;
+    input{
+      flex: 1;
+    }
+    i{
+      margin-left: -25px;
+      color: #247dc5
+    }
+  }
+  .btn {
+    &.btn-outline-primary{
+      background: none;
+      color: #247dc5;
+      border: 1px solid #247dc5;
+    }
+    &.clean-btn {
+      margin-right: 10px;
+
+      &:hover {
+        background-color: white;
+      }
+    }
+    &.search-btn{
+      margin: 0 !important;
```

Comment:
I understand previos changes were legacy code, but on new code, is there a way to workaround the !important ?